health/health.d/mysql.conf

```diff
@@ -160,7 +160,7 @@ component: MySQL
 component: MySQL
      calc: $wsrep_cluster_status
     every: 10s
-     crit: $mysql_galera_cluster_state != nan AND $this != 0
+     crit: $mysql_galera_cluster_state != nan AND $this != 0 AND $this != 'Primary'
```

Comment:
That is wrong, `$this` is an integer value. 

---

The current implementation is to decode the status as an integer, see the [function](https://github.com/netdata/go.d.plugin/blob/faa01a27a40818eeef5b849ae599353ab000dc48/modules/mysql/collect_global_status.go#L65-L84).

The alarm is critical when the wsrep cluster status is not 0 (not primary/quorum present).

Reply:
Maybe it's wrong but if I don't apply that change for non integer value I keep receiving an alert for this while the graph shows a flat 0. I don't say it's the right way to fix it but having no time to dig into source code I fixed it that way on 7 servers. 

So let me rephrase all this: I get a critical alarm as seen on screenshot if I don't make that change.

Reply:
The only screenshot in the issue is the Cloud Alarms tab screenshot. Can you share a screenshot from the local dashboard?

<details>
<summary>e.g. screenshot click</summary>

<img width="1213" alt="Screenshot 2022-08-22 at 15 44 45" src="https://user-images.githubusercontent.com/22274335/185924143-3ff1835e-f848-4352-90bb-7fef2fa54459.png">

</details>

Reply:
![critical_alert](https://user-images.githubusercontent.com/104206200/185925238-c833642c-b27a-4aaa-920a-26ac56c2ea06.png)

Reply:
Thanks, but I asked for a screenshot from the local dashboard, not Cloud.

Reply:
I don't use the local dashboard because access to those servers is simply not available.